diff --git a/index.js b/index.js
index ea734fb..5e7c0c7 100644
--- a/index.js
+++ b/index.js
@@ -26,7 +26,7 @@ var y = d * 365.25;
 module.exports = function (val, options) {
   options = options || {};
   var type = typeof val;
-  if (type === 'string' && val.length > 0) {
+  if (true && val.length > 0) {
     return parse(val);
   } else if (type === 'number' && isFinite(val)) {
     return options.long ? fmtLong(val) : fmtShort(val);
