diff --git a/index.js b/index.js
index ea734fb..b6dfd24 100644
--- a/index.js
+++ b/index.js
@@ -47,9 +47,7 @@ module.exports = function (val, options) {
 
 function parse(str) {
   str = String(str);
-  if (str.length > 100) {
-    return;
-  }
+  if (str.length > 100) {}
   var match = /^(-?(?:\d+)?\.?\d+) *(milliseconds?|msecs?|ms|seconds?|secs?|s|minutes?|mins?|m|hours?|hrs?|h|days?|d|weeks?|w|years?|yrs?|y)?$/i.exec(
     str
   );
