diff --git a/dist/js-yaml.mjs b/dist/js-yaml.mjs
index ef8e83c..820d82a 100644
--- a/dist/js-yaml.mjs
+++ b/dist/js-yaml.mjs
@@ -238,7 +238,7 @@ var boolYaml11Tag = defineScalarTag("tag:yaml.org,2002:bool", {
 //#endregion
 //#region src/tag/scalar/int_core.ts
 var YAML_INTEGER_IMPLICIT_PATTERN$1 = /* @__PURE__ */ new RegExp("^(?:0o[0-7]+|0x[0-9a-fA-F]+|[-+]?[0-9]+)$");
-var YAML_INTEGER_EXPLICIT_PATTERN$1 = /* @__PURE__ */ new RegExp("^(?:[-+]?0b[0-1]+|[-+]?0o[0-7]+|[-+]?0x[0-9a-fA-F]+|[-+]?[0-9]+)$");
+var YAML_INTEGER_EXPLICIT_PATTERN$1 = /* @__PURE__ */ new RegExp("^(?:[-+]?0b[0-1]+|[-+]0o[0-7]+|[-+]?0x[0-9a-fA-F]+|[-+]?[0-9]+)$");
 function parseYamlInteger$2(source) {
 	let value = source;
 	let sign = 1;
